db/migrate/20160827111345_add_latitude_and_longitude_to_drops.rb

```diff
@@ -0,0 +1,21 @@
+class AddLatitudeAndLongitudeToDrops < ActiveRecord::Migration
+
+  class Drop < ActiveRecord::Base
+    belongs_to :place
+  end
+
+  def up
+    add_column :drops, :latitude, :float
+    add_column :drops, :longitude, :float
+
+    Drop.reset_column_information
```

Comment:
ah, so this migration both adds new columns to the database AND populates those columns with lat & long values?